templates/address-space-controller/020-ConfigMap-address-space-definitions.yaml

```diff
@@ -359,6 +359,8 @@ data:
                 value: ${CONTROLLER_RESYNC_INTERVAL}
               - name: CHECK_INTERVAL
                 value: ${CONTROLLER_CHECK_INTERVAL}
+              - name: CONFIGMAP_WATCH_INTERVAL
```

Comment:
This variable does not seem to be used.

Reply:
KeycloakController reads environment variable `CONFIGMAP_WATCH_INTERVAL` and defaults it to 5 minutes if not set.   I think what is missing is a ` parameter` defaulting its value in the template.

Reply:
But this is the admin deployment, not the keycloak-controller deployment, which is located in templates/standard-authservice/050-Deployment-keycloak-controller.yaml